web/portal/src/main/webapp/WEB-INF/conf/organization/idm-configuration.xml

```diff
@@ -100,6 +104,16 @@
         <!--<value>war:/conf/organization/picketlink-idm/examples/picketlink-idm-msad-readonly-config.xml</value>-->
       </value-param>
 
+      <value-param>
+        <name>config.ldap</name>
+        <value>war:/conf/organization/picketlink-idm/picketlink-idm-ldap-config.xml</value>
+      </value-param>
+
+      <value-param>
+        <name>config.ad</name>
+        <value>war:/conf/organization/picketlink-idm/examples/picketlink-idm-msad-readonly-config.xml</value>
```

Comment:
Could you move this to the same path as picketlink-idm-ldap-config.xml ? it will be easier to check which files are used .

Reply:
Yes, I did not implement AD conf, it is in the TODO list. I will put files in the same folder.

Reply:
Done.